kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/Dispatch.java

```diff
@@ -218,6 +221,21 @@ public BaseEntity handleNonReadonly(ProcessData processData, Set<Ask> asks, Map<
 		return processEntity;
 	}
 
+	/**
+	 * Send the button events
+	 * @param asks
+	 */
+	public void sendButtonEvents(Set<Ask> asks) {
+		for (Ask ask : asks) {
+			if (ask.getQuestionCode().equals(Question.QUE_EVENTS)) {
```

Comment:
flip this equals
Question.QUE_EVENTS.equals(ask.getQuestionCode())

Reply:
done